src/ecosystem/Ecosystem.jl

```diff
@@ -45,3 +46,119 @@ function EnvironmentalLayer(
         ),
     )
 end
+
+"""
+    truncated_standard_normal_mean(a::Float64, b::Float64)::Float64
+
+Calculates the mean of the truncated standard normal distribution. Implementation taken
+from Distribution.jl excluding unused error checks
+
+# Arguements
+- 'a' : lower bound of the truncated distribution
+- 'b' : upper bound of the truncated distribution
+
+# Returns
+- Float64, mean of the truncated standard normal distribution
```

Comment:
No need to make this a dot point if there's only a single return value.

General Julia style recommendation is to reduce repetition where possible/appropriate. As the return type is already documented in the function signature above, there's no need to repeat it here.

```suggestion
The mean of the truncated standard normal distribution
```

Reply:
Go through and make the same changes for all the docstrings in this PR